include/cst_synth.h

```diff
@@ -45,51 +45,48 @@
 #include "cst_voice.h"
 #include "cst_wave.h"
 
-typedef int (*cst_breakfunc)(cst_tokenstream *ts, 
-		             const char *token, 
-			     cst_relation *tokens);
-CST_VAL_USER_FUNCPTR_DCLS(breakfunc,cst_breakfunc)
-int default_utt_break(cst_tokenstream *ts,
-		      const char *token, cst_relation *tokens);
+typedef int (*cst_breakfunc) (cst_tokenstream *ts,
+                              const char *token, cst_relation *tokens);
+CST_VAL_USER_FUNCPTR_DCLS(breakfunc, cst_breakfunc)
+     int default_utt_break(cst_tokenstream *ts,
```

Comment:
Maybe I fail to see something, but I think that the line after the `CST_VAL_USER_FUNCPTR_DCLS` should not be indented. Is it indented because the macro does not have a newline at the end or am I missing something?

Maybe adding an extra ";" after the macro would solve the indentation issue.